src/composables/multisigAccounts.ts

```diff
@@ -66,6 +68,12 @@ const activeMultisigAccountId = ref<AccountAddress>();
 const activeMultisigNetworkId = ref('');
 const isAdditionalInfoNeeded = ref(false);
 
+const activeMultisigAccountIdx = useStorageRef<number>(
```

Comment:
While extension is not active the status of the multisig vaults can be changed (by another signer for example), that will change the order of the vaults and that will lead to opening a different vault, than was chosen in the last session.